server/index.js

```diff
@@ -6,10 +6,12 @@ const router = require('./Route.js');
 const app = express();
 
 app.use(morgan('dev'));
+app.use(express.json());
 app.use(express.static(path.resolve(__dirname, '..', 'public')));
 app.set('port', 3003);
```

Comment:
In the future, you may want to include an environment variable as a conditional argument!